codice/ddf
# DDF-6008 - Enable Solr Highlighting

#### What does this PR do?

#### Who is reviewing it? 
@pklinef 
@vinamartin 

#### Select relevant component teams: 
@codice/solr 

#### Ask 2 committers to review/merge the PR and tag them here.

@andrewkfiedler
@jlcsmith


#### How should this be tested?
Submit a text query with the request property `highlight` set to true.
Verify that the response properties contain the `highlight` key and the value contains field information about the matched data within the fields that matched the query.

#### Any background context you want to provide?

#### What are the relevant tickets?
Fixes: #6008 

#### Screenshots


#### Checklist:

#### Notes on Review Process
Please see [Notes on Review Process](https://codice.atlassian.net/wiki/spaces/DDF/pages/71946981/Pull+Request+Guidelines) for further guidance on requirements for merging and abbreviated reviews. 

#### Review Comment Legend:
- ✏️ (Pencil) This comment is a nitpick or style suggestion, no action required for approval. This comment should provide a suggestion either as an in line code snippet or a gist. 
- ❓ (Question Mark) This comment is to gain a clearer understanding of design or code choices, clarification is required but action may not be necessary for approval.
- ❗ (Exclamation Mark) This comment is critical and requires clarification or action before approval.
